python/nvfuser/__init__.py

```diff
@@ -64,10 +64,10 @@ def __init__(
         id=None,
         max_length=9999,
         use_multidevice_executor=False,
-        backend_type=CommunicatorBackend.nccl,
+        multi_device_executor_params=MultiDeviceExecutorParams(),
```

Comment:
Since test_overlap.py already migrated to nvfuser_direct, I think you can leave this file as is.